### PR TITLE
Add recipe for inwx-api-bundle >= v1.0

### PR DESCRIPTION
**Pull request by SebTM**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: sebtm/inwx-api-bundle/1.0/config/packages/inwx_api.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Read the documentation: https://github.com/SebTM/inwx-api-bundle
+inwx_api:
+    debug: false
+    environment: development
```
Comment (Nyholm) [inline sebtm/inwx-api-bundle/1.0/config/packages/inwx_api.yaml]: You should remove these two and maybe read the default from `%kernel.debug%`, `%kernel.environment%`, right?

Comment (SebTM) [reply sebtm/inwx-api-bundle/1.0/config/packages/inwx_api.yaml]: I understand the first point with removing the not needed defaults, but I think there are cases you want to enable debugging and/or switch to different environment regardless "%kernel.*%" settings. I would suggest to set them to the "%kernel.*" as default but keep them independent changeable.

Would you be fine with this?

Comment (SebTM) [reply sebtm/inwx-api-bundle/1.0/config/packages/inwx_api.yaml]: @Nyholm Ping, would you be fine with this as proposed?

Comment (Nyholm) [reply sebtm/inwx-api-bundle/1.0/config/packages/inwx_api.yaml]: Yes

**Pull request merged**